.github/workflows/tests.yml

```diff
@@ -0,0 +1,23 @@
+name: CI Tests
+on: [push, pull_request]
```

Comment:
Same comment here, I think on: [pull_request] should be enough

Reply:
As discussed, we will keep push now and change it in a follow up to push on branch develop to avoid two runs for any branches created off the main repo. This will be in a follow up